lib/membrane_webm/schema/schema.ex

```diff
@@ -0,0 +1,339 @@
+           {"55B0", :Colour},
+           # \Segment\Tracks\TrackEntry\Video\Colour
+           {"55B7", :ChromaSitingHorz},
+           {"55B8", :ChromaSitingVert},
+           #               # Video Start
+           # AlphaMode
+           # PixelCropBottom
+           # PixelCropTop
+           # PixelCropLeft
+           # PixelCropRight
+           # DisplayWidth
+           # DisplayHeight
+           # DisplayUnit
+           # AspectRatioType
+           # Deprecated	FrameRate
+           #               # Video End
+           #               # Audio Start
+           # Audio
+           # SamplingFrequency
+           # OutputSamplingFrequency
+           # Channels
+           # BitDepth
+           #               # Audio End
+           #               # Content Encoding Start
+           # ContentEncoding
+           # ContentEncodingOrder
+           # ContentEncodingScope
+           # ContentEncodingType
+           # ContentEncryption
+           # ContentEncAlgo
+           # ContentEncKeyID
+           # ContentEncAESSettings
+           # AESSettingsCipherMode
+           #               # Content Encoding End
+           #               # Colour
+           # changing location of the colour element in file https://www.webmproject.org/docs/container/#LuminanceMin
+           # Colour
+           # MatrixCoefficients
+           # BitsPerChannel
+           # ChromaSubsamplingHorz
+           # ChromaSubsamplingVert
+           # CbSubsamplingHorz
+           # CbSubsamplingVert
+           ### ChromaSitingHorz
+           ### ChromaSitingVert
+           # Range
+           # TransferCharacteristics
+           # Primaries
+           # MaxCLL
+           # MaxFALL
+           # MasteringMetadata
+           # PrimaryRChromaticityX
+           # PrimaryRChromaticityY
+           # PrimaryGChromaticityX
+           # PrimaryGChromaticityY
+           # PrimaryBChromaticityX
+           # PrimaryBChromaticityY
+           # WhitePointChromaticityX
+           # WhitePointChromaticityY
+           # LuminanceMax
+           # LuminanceMin
+           #         # Chapters
+           # Chapters
+           # EditionEntry
+           # ChapterAtom
+           # ChapterUID
+           # ChapterStringUID
+           # ChapterTimeStart
+           # ChapterTimeEnd
+           # ChapterDisplay
+           # ChapString
+           # ChapLanguage
+           # ChapCountry
+           {"1549A966", :Info},
+           # \Segment\Info
+           {"2AD7B1", :TimecodeScale},
+           {"7BA9", :Title},
+           {"4D80", :MuxingApp},
+           {"5741", :WritingApp},
+           {"4489", :Duration},
+           {"4461", :DateUTC},
+           {"114D9B74", :SeekHead},
+           # \Segment\SeekHead
+           {"4DBB", :Seek},
+           # \Segment\SeekHead\Seek
+           {"53AC", :SeekPosition},
+           {"53AB", :SeekID}
+         ])
+
+  @element_info %{
+    EBML: %{type: :master},
```

Comment:
Do you need the maps? You can have a simple mapping from module to type itself. `@element_id_to_type_mapping` would be then more appropriate.